flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/executor/BatchExecutor.java

```diff
@@ -42,47 +43,108 @@
 @Internal
 public class BatchExecutor extends ExecutorBase {
 
+	private boolean enableObjectReuse;
```

Comment:
create a structure for these backed up variables?

Reply:
Created.